### ISSUE TITLE
Lang is null on some cases?

### ISSUE THREAD

**Issue by GreWull** [adonisjs/adonis-antl#11]
I'm trying to implement the multilanguage with the middleware but I've some problems.

First problem:
The route is defined as

`Route.on('/').render('index')`

When I go on **example.com/** it shows the correct default language.

When I go on **example.com/?lang=fr** it show again the default languange, but if I refresh the page it actually show the correct "fr" language.

Second (bigger) problem:

Routes:
```
Route.resource('Member', 'MemberController').middleware('auth', 'admin').except('index', 'show')
Route.get('/team', 'MemberController.index')
```
Here the language doesn't work at all. It always show the default language. I modified the middleware Antl.js to print the lang parameter and I get 3 prints, so the middleware it's called 3 times 

```
console.log(request.input('lang'))
// output in console
fr
null
null

```

This doesn't happen on another route like 
`Route.on('/foo').render('foo')`
In this case it prints only one time the lang.

I can't understand what's the problem!

**Comment by yeedavid** [comment]
Yes, I also found this issue. when Antl (en,cn,fr,ja,ru,etc) has been integrated in my local project, I have been keeping click each of locale to change the language, but it would be failed sometime unless double click the same locale. May someone to help to fix this issue or how to refresh the cache / view after changed the locale.

**Comment by thetutlage** [comment]
Going to check it now

**Comment by thetutlage** [comment]
Okay I just tried it with following code and it works fine for me

```
const Route = use('Route')
const Antl = use('Antl')

Route.get('/', function * (req, res) {
  res.send(Antl.getLocale())
})
Route.post('/', 'TestController.print')
```

Also you need to have locales saved inside `resources/locales` directory for `Antl` to show that language.

For example if you visit `example.com?lang=fr` and there is no `resources/locales/fr` directory. The Antl will fallback to `en-us`

Also I suggest adding `--watch resources/locales` to the `serve:dev` script inside `package.json` file, so that nodemon reloads the server when you make changes to this directory.

**Comment by GreWull** [comment]
The problem is still there.. But I've noticed that it only happens if I use Antl inside the View with
`{{ antl.get('messages.test') }}`

If I print the result inside the controller I actually get the right thing

`console.log(Antl.get('messages.test'))`

I could get all the string needed in the controller and the pass them to the view and it would work, but it would be better if I could use `{{ antl.get('messages.test') }}` in the view!

Any tips?

**Comment by thetutlage** [comment]
Yup reproduced the issue. I will work on this and will notify you

**Comment by thetutlage** [comment]
Fixed in 4.0
